### PR TITLE
fix(#3): add a workaround for stripped wasm symbols

### PR DESCRIPTION
**Pull request by piotr-roslaniec**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,6 +17,16 @@ env:
   RUSTFLAGS: "-Dwarnings"
 
 jobs:
+  build:
```
Comment (piotr-roslaniec) [inline .github/workflows/ci.yml:20]: Just to document the fix

Comment (fjarri) [reply .github/workflows/ci.yml:20]: I don't think it is necessary. The bug seems to be in `wasm-bindgen`, so we shouldn't add any regressions for it.

File: Cargo.toml [hunk 2]
```diff
@@ -3,3 +3,6 @@ members=[
     'wasm-bindgen-derive',
     'tests'
 ]
+
+[profile.release]
+opt-level = 1
```
Comment (fjarri) [inline Cargo.toml:8]: Please add an explanation and a link to https://github.com/rustwasm/wasm-bindgen/issues/3457 here.

I am still not quite sure how to reproduce the bug properly; when compiling `wasm-bindgen-derive`, it only occurs for a release build, but in my test above it happens in both release and debug builds (and in the failed `nucypher-core` CI it also happened in debug, namely during a `clippy` run)

**Comment by fjarri** [comment]
Small nitpick, you can just write "Fixes #3" in the PR description, that will link the issue, and close it automatically on merge.

Also, if I understand it correctly, adding `profile.release` only fixes the build of this crate, but not any dependent crates, right? So perhaps we should mention in the docs that the workaround is adding that `opt-level` in the release profile, and link the `wasm-bindgen` issue.

**Comment by fjarri** [comment]
Closing in favor of #5

**Pull request closed without merging**